engine/src/test/java/io/camunda/zeebe/engine/util/JobWorkerElementBuilderProvider.java

```diff
@@ -30,6 +31,14 @@ public Stream<? extends Arguments> provideArguments(final ExtensionContext exten
     return builders().map(Arguments::of);
   }
 
+  public static List<BpmnElementType> getSupportedActivities() {
+    return List.of(
+        BpmnElementType.SERVICE_TASK,
+        BpmnElementType.BUSINESS_RULE_TASK,
+        BpmnElementType.SCRIPT_TASK,
+        BpmnElementType.SEND_TASK);
```

Comment:
🔧 I'm a bit confused about this list of tasks. This is the `JobWorkerElementTest`. A `BUSINESS_RULE_TASK` and a `SCRIPT_TASK` don't create jobs. Meanwhile, `USER_TASK` does but it's not part of this list.

Reply:
👍 Business rule task and script task have multiple implementations. Users can choose the implementation in Modeler. If they choose Job Worker, then they behave as a service task, which allows users to use their own script engine or their own business rule processor.

🤔 I'm also surprised to see that user task is missing. The reason is that it does not implement the same model, i.e. users cannot define the job type. It's just an implementation choice that we've implemented user tasks with a job for now.

Reply:
Ah that makes sense. Thanks for the explanation 👍